internal/driver/glfw/window.go

```diff
@@ -237,7 +237,7 @@ func (w *window) ShowAndRun() {
 
 // Clipboard returns the system clipboard
 func (w *window) Clipboard() fyne.Clipboard {
-	return &clipboard{}
```

Comment:
Rather than allocating a new clipboard, should we just return the singleton instance from the app here?